p/p7zip/stone.yaml

```diff
@@ -0,0 +1,24 @@
+name: 7zip
+version: 24.09
+release: 1
+summary: 7-Zip command-line version for Linux
+license: LGPL-2.1-or-later and BSD 3-clause License
```

Comment:
That's not how you add a multiple licensed package,

BSD-3-Clause 
On a new line and without and, and license

Reply:
As it must be formatted like the identifiers in this list
https://spdx.org/licenses/